ql/src/java/org/apache/hadoop/hive/ql/metadata/HiveStorageHandler.java

```diff
@@ -638,4 +640,10 @@ default void prepareAlterTableEnvironmentContext(AbstractAlterTableDesc alterTab
   default Boolean hasAppendsOnly(org.apache.hadoop.hive.ql.metadata.Table hmsTable, SnapshotContext since) {
     return null;
   }
+
+  default List<String> showPartitions(DDLOperationContext context,
```

Comment:
please add a javadoc, this is an interface

Reply:
done